portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,14 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomPlayer() {
```

Comment:
It's a good habit to add a docstring describing this function.

Reply:
Here is the javascript style guide: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments